qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -18,6 +26,12 @@ You can download an example sample information file and prep information file fr
 Sample information file
 -----------------------
 
+Similar to a `QIIME metadata file, <http://qiime.org/documentation/file_formats.html#metadata-mapping-files>`__
```

Comment:
Thanks @colinbrislawn we really appreciate your input in these! I think user-written documentation is something that we never do and it can be a game changer to actually improve the quality of the docs, avoiding the use of technicalities that only the developers understand. Thanks!